Sources/Support/SDKTester.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SDKTester.swift
+//
+//  Created by Nacho Soto on 9/21/22.
+
+import Foundation
+
+/// `SDKTester` allows you to ensure that the SDK is set up correctly by diagnosing configuration errors.
+/// To run the test, simply call ``SDKTester/testRevenueCatIntegrationWithCompletion()``.
+///
+/// #### Example:
+/// ```swift
+/// let tester = SDKTester.default
+/// do {
+///     try await tester.testRevenueCatIntegration()
+/// } catch {
+///     print("SDKTester failed: \(error.localizedDescription)")
+/// }
+/// ```
+@objc(RCSDKTester)
+@available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
+public final class SDKTester: NSObject {
```

Comment:
I kind of think that naming this class more like a support or diagnostics tool, like `RevenueCatDiagnostics` or `PurchasesDiagnostics` as it more indicates symptoms of a potentially broken implementation. I think it would make it less ambiguous too, as the `test` terminology might be confusing/ pointing toward it being an internal thing rather than publicly accessible

Reply:
I like `PurchasesDiagnostics` 👍🏻